### DISCUSSION TITLE
Nextjs app with react spa under the same custom domain with subdomains

### DISCUSSION THREAD

**Discussion by flrnd** [vercel/vercel#5307] [Help]
I have a nextjs and a react spa both hosted at vercel. I'd like to have both under the same domain using subdomains 
 where  `example.com/routes`  are for the nextjs app and `myapp.example.com` for the react spa. I've been lurking the documentation and previous questions but I'm not sure if that's possible at all.

Is there any way to achieve this? If not, what other alternatives do I have? Thanks!

**Comment by adamaslan** [comment]
I think I am doing the same thing as you and also struggling. My main domain is created thru next.js and subdomain is created thru create react app. You have something similar going on? Have you made any progress? 
this is my error:
Error: No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies"
